care/abdm/api/viewsets/consent.py

```diff
@@ -71,19 +69,9 @@ def create(self, request):
 
         consent = ConsentRequest(**serializer.validated_data, requester=request.user)
 
-        try:
-            response = Gateway().consent_requests__init(consent)
-            if response.status_code != 202:
-                return Response(response.json(), status=response.status_code)
-        except Exception as e:
-            logger.warning(
-                f"Error: ConsentViewSet::create failed to notify (consent_requests__init). Reason: {e}",
-                exc_info=True,
-            )
-            return Response(
-                {"detail": "Failed to initialize consent request"},
-                status=status.HTTP_400_BAD_REQUEST,
-            )
+        response = Gateway().consent_requests__init(consent)
+        if response.status_code != 202:
+            return Response(response.json(), status=response.status_code)
```

Comment:
we had issues with abdm not actually responding with josn, we catch jsondecodeerror